src/write.c

```diff
@@ -41,20 +41,15 @@ static void writeConfigBox(avifRWStream * s, avifCodecConfigurationBox * cfg);
 // ---------------------------------------------------------------------------
 // avifSetTileConfiguration
 
-static int countLeadingZeros(uint32_t n)
+static int floorLog2(uint32_t n)
 {
-    int count = 32;
+    assert(n > 0);
```

Comment:
One comparison can be saved by using a `do {} while (n != 0)` thanks to this assertion.

Reply:
You are right. I will keep this form because I found that it is essentially the same as the ` FloorLog2()` function in the AV1 spec. Also it is convenient to define `floorLog2(0)` as -1. This implementation supports that if we remove the assertion.